test/requests/hover_expectations_test.rb

```diff
@@ -247,9 +247,12 @@ class Post
       })
 
       assert_nil(response.error, response.error&.full_message)
-      assert_match(<<~RESPONSE.strip, response.response.contents.value)
+      assert_equal(<<~RESPONSE.strip, response.response.contents.value)
+        ```ruby
+        Post
         Title
         Signature
+        ```
```

Comment:
Because of the restructuring, the test was complaining if "```" was not embedded. I decided to restructure the test to cover the whole hover.

Overall, this test isn't as comprehensive as the hardcoded fixtures anyways, and those have not been changed